odalic/src/main/java/cz/cuni/mff/xrg/odalic/outputs/annotatedtable/DefaultResultToAnnotatedTableAdapter.java

```diff
@@ -57,24 +61,24 @@ public AnnotatedTable toAnnotatedTable(Result result, Input input, Configuration
     return new AnnotatedTable(input.identifier(), new TableSchema(columns));
   }
   
-  private TableColumn createOriginalColumn(String columnName) {
+  private TableColumn createOriginalColumn(String columnName, boolean chosenIsEmpty) {
     return new TableColumn(columnName, Arrays.asList(columnName),
-        "", "anyURI",
+        "", chosenIsEmpty ? "" : "anyURI",
         false, false,
-        bracketFormat(urlFormat(columnName)), "", "dcterms:title", "");
+        chosenIsEmpty ? "" : bracketFormat(urlFormat(columnName)), "", chosenIsEmpty ? "" : "dcterms:title", "");
```

Comment:
Previously suggested note https://github.com/odalic/sti/pull/112/files#r74633920 will make these disappear, but even then try to limit the use of ternary operators as much as possible. Multiple ternary operators at one place indicate a code smell.